k/kernel-headers-system-docker.yml

```diff
@@ -0,0 +1,9 @@
+kernel-headers-system-docker:
+  image: rancher/os-headers:v0.4.0-dev
+  labels:
+    io.rancher.os.detach: false
+    io.rancher.os.after: network
+    io.rancher.os.scope: system
+  volumes:
+  - /usr/src:/usr/src
```

Comment:
For non-dev builds this should survive reboot.  For dev builds we delete /usr in the system root on each boot.